service/src/main/java/uk/nhs/adaptors/gp2gp/ehr/utils/TextUtils.java

```diff
@@ -0,0 +1,22 @@
+package uk.nhs.adaptors.gp2gp.ehr.utils;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import org.apache.commons.lang3.StringUtils;
+
+public class TextUtils {
+    public static String newLine(Object... values) {
+        return StringUtils.joinWith(StringUtils.LF, values);
```

Comment:
Should we worry at all about our host system?
```suggestion
        return StringUtils.joinWith(System.lineSeparator(), values);
```